Misc/NEWS.d/next/Library/2018-04-22-18-27-13.bpo-33338.185_om.rst

```diff
@@ -1,8 +1,6 @@
-lib2to3's tokenizer was synchronized with the built-in tokenizer, providing
-many wins, including:
-
-- the ``tokenize.tokenize()`` API is now mimicked in ``lib2to3``, including
-automatic encoding detection and yielding ENCODING tokens; - Unicode
-identifiers are now supported; - ELLIPSIS is its own token type now; -
-Untokenizer improved with backports of 5e6db31, 9dc3a36, 5b8d2c3, e411b66,
-and BPO-2495.
```

Comment:
The problem was:

>Warning, treated as error:
>../build/NEWS:137:Bullet list ends without a blank line; unexpected unindent.

which I think means that this is the correct rst format:

```
- bla bla item one
  second line is indentend

- bla bla second item after blank line
```